lib/checks/canMultiplyLikeTermsNthRoots.js

```diff
@@ -0,0 +1,19 @@
+const Node = require('../node');
+
+function canMultiplyNthRoots(node) {
+  // checks if node is a multiplication of nthRoot nodes
+  // all the terms has to have the same root node to be multiplied
```

Comment:
can you put docstrings above the function names? and add an example of some math that would be handled here